openapi/pdfbroker/openapi.yml

```diff
@@ -0,0 +1,698 @@
+openapi: 3.0.1
+info:
+  title: PdfBroker.io API
+  x-init-description: >- 
+    The connector initialization requires setting the API credentials.
+    Create an [PdfBroker account](https://www.pdfbroker.io/) and obtain tokens following 
```

Comment:
Please add a new line before this. You can check what was happened in the generated code init doc comment.